packages/itwinui-react/src/core/Menu/MenuItem.tsx

```diff
@@ -106,7 +106,7 @@ export const MenuItem = React.forwardRef<HTMLLIElement, MenuItemProps>(
     const [isSubmenuVisible, setIsSubmenuVisible] = React.useState(false);
 
     const onKeyDown = (event: React.KeyboardEvent<HTMLLIElement>) => {
-      if (event.altKey) {
+      if (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey) {
```

Comment:
What you're doing here is adding handling CTRL, meta key and SHIFT key. What we need to do is adding **only** ALT key handling. Meaning this was already handled before your edit, or it was already good/how it should be. So you should revert or remove your change here. Instead, find any `onKeyDown` function that **do not** have ALT key handling yet, and then add ALT key handling to it.

For example, check out the `onKeyDown` function of file/component `StepperStep.tsx` which is defined on line 65. You can see that within this function, there is no ALT key handling:

```
  const onKeyDown = (e: React.KeyboardEvent) => {
    if (!isClickable) {
      return;
    }

    if (e.key === 'Enter' || e.key === 'Space' || e.key === ' ') {
      onCompletedClick();
    }
  };
 ```
 
 You can add ALT key handling into this function by adding condition to check for ALT key as follows:
 ```
   const onKeyDown = (e: React.KeyboardEvent) => {
    if (e.altKey){
      return;
    }
    
    if (!isClickable) {
      return;
    }

    if (e.key === 'Enter' || e.key === 'Space' || e.key === ' ') {
      onCompletedClick();
    }
  };
  ```
 
And that is all you need to do. You should look for all `onKeyDown` functions that do not have ALT key handling yet, and add the ALT key handling into it like my example above. To start with, you can check out `StepperStep.tsx` and imitate my example.